Tasks/DownloadBuildArtifacts/task.json

```diff
@@ -30,30 +30,55 @@
       "type": "pickList",
       "label": "Build definition",
       "defaultValue": "",
-      "required": true,
+      "required": false,
       "properties": {
         "EditableOptions": "True"
       },
       "helpMarkDown": "Select the build definition name"
     },
     {
-      "name": "version",
+      "name": "buildId",
       "type": "pickList",
       "label": "Build",
-      "defaultValue": "",
+      "defaultValue": "$(Build.BuildId)",
       "required": true,
       "properties": {
         "EditableOptions": "True"
       },
       "helpMarkDown": "Select the build number"
     },
     {
+      "name": "downloadType",
+      "type": "radio",
+      "label": "Download Type",
+      "defaultValue": "single",
```

Comment:
Shouldn't the default be "all artifacts"?

Reply:
Current task implementation downloads only single artifacts, having default as single is to maintain compact.